packages/sdk/src/execute/execute.utils.ts

```diff
@@ -0,0 +1,184 @@
+import type { Wallet, FinalExecutionOutcome } from '@near-wallet-selector/core';
```

Comment:
Wouldn't this be `execute/utils.ts` instead of `execute/execute.utils.ts`?

Reply:
this is to keep naming convention as the tests we add the name of the method before too like transfer.test.ts